### PR TITLE
Fold the function call itself if it's stateless.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Follow https://github.com/vesoft-inc/nebula-common/pull/281

### CODE REVIEW

File: src/visitor/FoldConstantExprVisitor.cpp [hunk 1]
```diff
@@ -75,6 +77,14 @@ void FoldConstantExprVisitor::visit(FunctionCallExpression *expr) {
             }
         }
     }
+    auto result = FunctionManager::getStateless(*expr->name(), expr->args()->args().size());
```
Comment (yixinglu) [inline src/visitor/FoldConstantExprVisitor.cpp:80]: prefer `sideEffect` or `isPure` than `stateless` when we talk about function

Comment (dutor) [reply src/visitor/FoldConstantExprVisitor.cpp:80]: > prefer `sideEffect` or `isPure` than `stateless` when we talk about function

Agreed. `isPure` is better.

Comment (Shylock-Hg) [reply src/visitor/FoldConstantExprVisitor.cpp]: Ok, fixed.

**Review by dutor — APPROVED** [review]
Great fix!

**Review by yixinglu — APPROVED** [review]

**Pull request merged**